examples/realtime-positioning/proposal.js

```diff
@@ -1,13 +1,9 @@
 var sensor = new GeolocationSensor({ accuracy: "low" });
 sensor.start();
-sensor.onchange = function(event) {
-    var coords = [event.reading.latitude, event.reading.longitude];
-    updateMap(null, coords, event.reading.accuracy);
+sensor.onreading = function() {
+    var coords = [sensor.latitude, sensor.longitude];
+    updateMap(null, coords, sensor.accuracy);
 };
 sensor.onerror = function(error) {
     updateMap(error);
 };
-sensor.onstatechange = function(e) {
-    console.log("STATE:", this.state)
```

Comment:
I see you are removing this, but <this> would be wrong here... why not move them all to array methods?

Reply:
PRs for further enhancements welcome :-)